cloudflare-analytics.sh

```diff
@@ -36,46 +36,61 @@ cloudflareapikey="${CLOUDFLARE_API_TOKEN:="dummy_api_key"}"
 cloudflarezoneid="${CLOUDFLARE_ZONE_ID:="dummy_zone_id"}"
 cloudflarezone="${CLOUDFLARE_ZONE_NAME:="dummy_zone_name"}"
 
+TMP_FILE_LASTRUN_TIME="${cloudflarezone}_lastrun.tmp"
+TMP_FILE_POSTFIX="${cloudflarezone}_data.$$.tmp"
+TMP_FILE_POSTFIX="${cloudflarezone}_data.tmp"
+
+CF_GQL_RESULTS_LIMIT=10000
+CF_GQL_SINCE_MINS="-419"
+CF_GQL_MIN_PERIOD=30 # Grab additional 30mins to cover lag CF data availability
+
+CF_DEFAULT_SINCE_MIN="-10080"
+CF_DEFAULT_UNTIL_MIN="0"
+
 ######################################
 # Define this to be "echo" in ENV to switch off non-observer operations.
 : "${ECHO:=""}"
 
-function process_results {
+#####################################################
+# Processing Analytics API results.
```

Comment:
Is it work stating params in the block comment - there's 2 here - say what they mean perhaps?

Reply:
Actually, the seance params should. be removed as it isn't used. But you missed that right? ;-)

Reply:
Which is now done.

Reply:
Think this should be something other than output here.

Reply:
We've discussed this - going to set it to resolved for now.